calculator/src/cli.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/local/bin/python3
+from src.lexer import lex
+from src.parser import rpn, infix
+
+
+def main():
```

Comment:
This looks good, but I'd like it if you could use the calculator by piping input into it from another program.  For instance, in the shell, can you run:
```
echo "1 + 1" | calc
```
and have it print out `2`?

Reply:
It appears this isn't going to work after looking at it with you.  I want you to do the following:
1. Catch the `EOFError` that is thrown by `input()`
2. Use [optparse](https://docs.python.org/2/library/optparse.html) to allow `-q` or `--quiet` in which you do not provide the prompt.

Reply:
👍